Source/WebKit/webpushd/WebPushDaemon.mm

```diff
@@ -53,6 +53,9 @@
 #import <UIKit/UIApplication.h>
 #endif
 
+#define WEBPUSHDAEMON_RELEASE_LOG(channel, fmt, ...) RELEASE_LOG(channel, "%{public}s [connection=%p, app=%{public}s]: " fmt, __func__, &connection, connection.subscriptionSetIdentifier().debugDescription().utf8().data(), ##__VA_ARGS__)
```

Comment:
`PUBLIC_LOG_STRING`?

Reply:
Since this is a .mm with no chance of ever becoming platform-independent, this file is already just using `{public}`, `{sensitive}` instead of the macros.